src/app/components/phys-desc/phys-desc.component.scss

```diff
@@ -0,0 +1,53 @@
+.phyDesc {
+    margin-bottom: 2rem;
+}
+
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.flex-item-label {
+    flex-basis: 100%;
+    margin-bottom: 0.1rem;
+}
+
+.flex-item-information {
+    margin-bottom: 0.7rem;
```

Comment:
Add `flex-basis:100%` and 
`
::ng-deep p {
        margin-bottom: 0;
    }
`